mine_extractor_refactored/models/data_models.py

```
@@ -0,0 +1,109 @@
+@dataclass
```

Ruff (0.12.2)

Undefined name `dataclass`

(F821)